lib/ModuleReason.js

```diff
@@ -4,9 +4,45 @@
 */
 "use strict";
 
-module.exports = class ModuleReason {
+const util = require("util");
+
+class ModuleReason {
 	constructor(module, dependency) {
 		this.module = module;
 		this.dependency = dependency;
+		this._chunks = null;
```

Comment:
Btw, it may seem weird to set `this._chunks = null` in the constructor, but it helps `v8` optimize object creation and mutation if it has a predictable shape.